exercises/practice/rotational-cipher/.meta/example.cpp

```diff
@@ -0,0 +1,22 @@
+#include "rotational_cipher.h"
+
+#include <sstream>
+
+namespace rotational_cipher {
+    std::string  rotate(const std::string& text, int shiftKey) {
+
+        std::ostringstream encoded_text;
```

Comment:
style nitpick: "lowerCamelCase" for the parameter but "snake_case" for the variable. Is that intentional?